Samples/IntelliTect.TestTools.Data.Sample/IntelliTect.TestTools.Data.Sample.sln

```diff
@@ -0,0 +1,22 @@
+﻿
```

Comment:
Why add another solution? I would expect the sample project to be pair of the main solution.